components/brave_wallet_ui/components/desktop/views/crypto/index.tsx

```diff
@@ -127,7 +127,7 @@ export const CryptoView = ({ sessionRoute }: Props) => {
     if (isPanel) {
       chrome.tabs.create(
         {
-          url: `chrome://wallet${WalletRoutes.Backup}`
+          url: `chrome://wallet${encodeURIComponent(WalletRoutes.Backup)}`
```

Comment:
`WalletRoutes.Backup` is also a hardcoded paths here https://github.com/brave/brave-core/pull/27636#discussion_r1957432756

Reply:
Good catch, I missed these ones previously.